models/bases/contributing_death_causes.sql

```diff
@@ -0,0 +1,11 @@
+select
+    cdc.id,
+    cdc.time_after_onset,
+    cdc.patient_death_data_id,
+    cdc.condition_id,
+    cdc.created_at
```

Comment:
Preference not to use metadata.
```suggestion
    cdc.created_at
```